servicetalk-http-api/src/main/java/io/servicetalk/http/api/SingleAddressHttpClientBuilder.java

```diff
@@ -50,6 +50,16 @@
  * @param <R> the type of address after resolution (resolved address)
  */
 public interface SingleAddressHttpClientBuilder<U, R> extends HttpClientBuilder<U, R, ServiceDiscovererEvent<R>> {
+    /**
+     * Configure proxy to serve as an intermediary for requests.
+     * @param proxyAddress Unresolved address of the proxy.
```

Comment:
Should we clarify here what should be used when a builder is created for resolved address?

Reply:
Makes sense. Although it's an implementation choice so the wording will include "may" instead of "will" for the note about runtime exceptions.